src/waveform-overview.js

```diff
@@ -344,6 +344,10 @@ define([
     this._playheadLayer.showPlayheadTime(show);
   };
 
+  WaveformOverview.prototype.setPlayheadTimePrecision = function() {
+    // The overview waveform doesn't show time codes for play head and markers
```

Comment:
This method can apply to the overview waveform, as the user can call `showPlayheadTime(true)`.

Reply:
Oh okay. I'll change it along with the README.